src/app/core/common-components/entity-form/form-field/form-field.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input, OnChanges, SimpleChanges } from "@angular/core";
+import { DynamicComponentDirective } from "../../../config/dynamic-components/dynamic-component.directive";
+import { HelpButtonComponent } from "../../help-button/help-button.component";
+import { Entity } from "../../../entity/model/entity";
+import { EntityForm, EntityFormService } from "../entity-form.service";
+import { ColumnConfig } from "../../entity-subrecord/entity-subrecord/entity-subrecord-config";
+import { FormFieldConfig } from "../entity-form/FormConfig";
+import { NgIf } from "@angular/common";
+
+/**
+ * Generic component to display one form field (editComponent) of an entity.
+ *
+ * Dynamically extends field details from entity schema and loads the relevant, specific EditComponent implementation.
+ */
```

Comment:
Thanks for pointing this out! I knew we had something for the ViewComponent but just couldn't find it ...

Should we have a single component that supports view + edit modes via input switch - or keep the two completely separate components? (i.e. keeping this implementation and deleting EntityPropertyViewComponent; Or removing the view mode here and using EntityPropertyViewComponent for that part).
What do you think, @TheSlimvReal ?

Reply:
Looking further into special cases in subrecord and rowdetails, where we use viewComponents as fallbacks or alternatives to editComponents, I am tending towards a single integrated component that handles view and edit mode.

Reply:
On the other hand we currently only have one place where edit and view components are switched (entity subrecord inline edit) which I don't think is a very useful mode. In all other cases there is quite a clear cut between edit components and view component. Therefore I would like to make the view component as simple as possible and keep to code a little more obvious. Otherwise its always quite difficult to understand why in a certain place you are supposed to use a form field component when you just want to display that value.

Reply:
That is also a valid point. implementation itself is probably cleaner and simpler if we keep this separated ...

Currently Subrecord and RowDetails use the view-mode switch. But they could both instead use the other component, passing fieldConfig to that to render the viewComponent.

Only one limitation: subrecord in inline edit mode is doing a fallback, showing viewComponents for those columns that do not have a editComponent. And for that again would need the full fieldConfig :roll_eyes: 
Alternatively: let FormFieldComponent use the "ViewFieldComponent" (for view mode) as a fallback but have that as a separated component?

Reply:
Ah I see. Then I think go ahead with the approach that you currently feel most comfortable with.

Reply:
I will clean this up now:
- FormFieldComponent will only handle edit mode
- ViewFieldComponent will follow the same pattern and handle view mode (renaming + moving EntityPropertyViewComponent for this)